docs/toc.json

```diff
@@ -92,6 +92,9 @@
 					},
 					{
 					  "docs/how-to-guides/data-basics/2-building-a-list-of-pages.md": []
+					},
```

Comment:
Part 3 is missing here, https://github.com/WordPress/gutenberg/pull/39387 adds it